orm/internal/testpb/test_schema.proto

```diff
@@ -9,68 +9,68 @@ import "cosmos/orm/v1/orm.proto";
 message ExampleTable {
   option (cosmos.orm.v1.table) = {
     id: 1;
-    primary_key: {
```

Comment:
can these spacing changes be reverted

Reply:
Alright, reverted to use `make proto-gen` instead. 👍🏾